tests/stdlib/trstgen.nim

```diff
@@ -702,12 +739,32 @@ Test1
       #. string2
       #. string3
       """
-    let output7 = rstToHtml(input7, {roSupportMarkdown}, defaultConfig())
+    let output7 = input7.toHtml
     doAssert count(output7, "<ol ") == 1
     doAssert count(output7, "</ol>") == 1
     doAssert count(output7, "<li>") == 3
     doAssert "start=\"3\"" in output7 and "class=\"upperalpha simple\"" in output7
 
+    # check that it's not recognized as enum.list without indentation on 2nd line
+    let input8 = dedent """
+      Paragraph.
+
+      A. stringA
+      B. stringB
+      C. string1
+      string2
+      """
+    let output8 = input8.toHtml(
+        expectWarnings = @["input(6, 0) Warning: RST style: \n" &
```

Comment:
not DRY because you're repeating verbatim the error message, which makes it more difficult to change the error message in future work as it'd break such tests; tests should be strict about detecting when some condition triggers but loose regarding what specific error/warning is produced. Instead, use something like:

```nim
doAssert "not enough indentation on line 6" in output 
```
or since you're using a helper, turn `expectWarnings` into a substring match

Reply:
fixed